src/main/java/org/sagebionetworks/bridge/hibernate/StudyIdentifierConverter.java

```diff
@@ -0,0 +1,19 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import org.sagebionetworks.bridge.models.studies.StudyIdentifier;
+import org.sagebionetworks.bridge.models.studies.StudyIdentifierImpl;
+
+@Converter
+public class StudyIdentifierConverter implements AttributeConverter<StudyIdentifier, String> {
```

Comment:
Unit tests for this class?

Reply:
Added. I'm not sure these converters receive null values, but I've added code so they behave in this case.